src/store/channels/saga.ts

```diff
@@ -45,35 +44,44 @@ export function* markAllMessagesAsRead(channelId, userId) {
 }
 
 // mark all messages as read in current active channel (only if you're not in full screen mode)
-export function* markChannelAsReadIfActive(channelId) {
-  const activeChannelId = yield select(activeChannelIdSelector);
-  if (channelId !== activeChannelId) {
-    return;
-  }
-
+export function* markChannelAsRead(channelId) {
   const currentUser = yield select(currentUserSelector());
   const isMessengerFullScreen = yield select((state) => state.layout.value.isMessengerFullScreen);
-  const activeChannelInfo = activeChannelId ? yield select(rawChannelSelector(activeChannelId)) : null;
+  const activeChannelInfo = yield select(rawChannelSelector(channelId));
```

Comment:
```suggestion
  const channelInfo = yield select(rawChannelSelector(channelId));
```

It's not the active channel.